src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -44,6 +44,8 @@ public string ClientVersion
             internal set { _clientVersion = string.IsNullOrWhiteSpace(value) ? DefaultClientVersion : value; }
         }
 
+        public ITelemetryClient[] TelemetryClients { get; internal set; }

```

Comment:
>public ITelemetryClient[] TelemetryClients { get; internal set; } [](http://example.com/codeflow?start=8&length=65)

init to Empty set? having a public setter will let any caller modify the set of telemetry clients, is that intentional?

What's the library stance on property headers?

Reply:
Actually changed this on the other PR and pulled in changes from there. So this is updated and initialized.